python/mxnet/kvstore_server.py

```diff
@@ -72,12 +72,12 @@ def run(self):
         _ctrl_proto = ctypes.CFUNCTYPE(None, ctypes.c_int, ctypes.c_char_p, ctypes.c_void_p)
         check_call(_LIB.MXKVStoreRunServer(self.handle, _ctrl_proto(self._controller()), None))
 
-def _init_kvstore_server_module():
+def _init_kvstore_server_module(data_type='float32'):
     """Start server/scheduler."""
     is_worker = ctypes.c_int()
     check_call(_LIB.MXKVStoreIsWorkerNode(ctypes.byref(is_worker)))
     if is_worker.value == 0:
-        kvstore = create('dist')
+        kvstore = create('dist', data_type)
```

Comment:
How do you start the server with float16 dtype?

Reply:
In image classification example, we start kvstrore as below.
In file 'common/fit.py'
```
    mx.kvstore_server._init_kvstore_server_module(args.dtype)
    kv = mx.kvstore.create(args.kv_store, args.dtype)
```
In file 'python/mxnet/kvstore_server.py', we delete ‘_init_kvstore_server_module()’ to avoid auto start kvstore server during import mxnet.
```
#_init_kvstore_server_module()
```